icicle/include/icicle/curves/projective.h

```diff
@@ -178,13 +179,32 @@ class Projective
 
   friend HOST_DEVICE Projective operator*(SCALAR_FF scalar, const Projective& point)
   {
+    // Precompute points: P, 2P, ..., (2^window_size - 1)P
+    constexpr unsigned window_size =
+      4; // 4 seems fastest. Optimum is minimizing EC add and depends on the field size. for 256b it's 4.
```

Comment:
Should it be part of the curve config? I assume the difference between 256 and 384 curves is significant

Reply:
I used this and both are optimized for 4:

2^w + (256/w) * (2^w-1) / (2^w)
2^w + (384/w) * (2^w-1) / (2^w)

w - the window bit size